lib/raidenclient/RaidenClient.ts

```diff
@@ -285,9 +292,26 @@ class RaidenClient extends SwapClient {
 
   /**
    * Creates a payment channel.
+   */
+  public openChannel = async (peerAddress: string, amount: number, currency: string): Promise<void> => {
+    const tokenAddress = this.tokenAddresses.get(currency);
+    if (!tokenAddress) {
+      throw(errors.TOKEN_ADDRESS_NOT_FOUND);
+    }
+    await this.openChannelRequest({
+      partner_address: peerAddress,
+      token_address: tokenAddress,
+      total_deposit: amount,
+      // TODO: The amount of blocks that the settle timeout should have
+      settle_timeout: 500,
+    });
+  }
+
+  /**
+   * Creates a payment channel request.
    * @returns The channel_address for the newly created channel.
    */
-  public openChannel = async (payload: OpenChannelPayload): Promise<string> => {
+  public openChannelRequest = async (payload: OpenChannelPayload): Promise<string> => {
```

Comment:
I think it would make sense to just combine this with the `openChannel` method above, although if you'd prefer not to I figure this method could at least be private.

Reply:
Thanks for catching this. I changed the method to be private.